decidim-meetings/app/controllers/decidim/meetings/registrations_controller.rb

```diff
@@ -32,9 +32,9 @@ def answer
       def create
         enforce_permission_to(:register, :meeting, meeting:)
 
-        @form = JoinMeetingForm.from_params(params)
+        @form = JoinMeetingForm.from_params(params, user: current_user)
```

Comment:
This is not how params and the forms work, you need to pass it as a context.

```suggestion
        @form = JoinMeetingForm.from_params(params).with_context(current_user:)
```

This is how it was done in #12761 for instance